Sources/StreamChatUI/ChatMessageList/Attachments/GiphyAttachmentViewInjector.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public typealias GiphyAttachmentViewInjector = _GalleryAttachmentViewInjector<NoExtraData>
+
+public class _GiphyAttachmentViewInjector<ExtraData: ExtraDataTypes>: _AttachmentViewInjector<ExtraData> {
+    /// A custom `UIImageView` view that always returns max. `intrinsicContentSize`.
+    private class EagerImageView: UIImageView {
+        override var intrinsicContentSize: CGSize { .init(width: .max, height: .max) }
+    }
```

Comment:
Can we maybe have it as an internal type? 🤔 ATM it is copy-pasted in gallery/giphy injectors. Then we can also set the `width=height` constraint inside.

Reply:
@evsaev this is just a temp workaround, we will remove it once we start using "real" views for attachments